Vectors.Rmd

```diff
@@ -754,77 +754,74 @@ rownames(df3)
 ```
 
 ### Printing 
 
-One of the most obvious differences between tibbles and data frames is how they are printed. I assume that you're already familiar with how data frames are printed, so here I'll highlight some of the biggest differences using an example dataset included in the dplyr package:
+One of the most obvious differences between tibbles and data frames is how they print. I assume that you're already familiar with how data frames are printed, so here I'll highlight some of the biggest differences using an example dataset included in the dplyr package:
 
 ```{r}
 dplyr::starwars
 ```
 
-Tibbles:
-
-* Only show the first 10 rows and all the columns that will fit on screen.
-  Additional columns are shown at the bottom.
+* Tibbles only show the first 10 rows and all the columns that will fit on 
+  screen. Additional columns are shown at the bottom.
 
 * Each column is labelled with its type, abbreviated to three or four letters.
 
 * Wide columns are truncated to avoid a single long string occupying an entire
-  row. (This is still a work in progress: it's tricky to get the tradeoff right 
+  row. (This is still a work in progress: it's tricky to get the tradeoff 
   between showing as many columns as possible and showing a single wide 
-  column fully.)
+  column in its entirety just right.)
 
 * When used in console environments that support it, colour is used judiciously 
   to highlight important information, and de-emphasise supplemental details.
 
 ### Subsetting {#safe-subsetting}
-
-As you will learn in Chapter \@ref(subsetting), you can subset a data frame or a tibble like a 1d structure (where it behaves like a list), or a 2d structure (where it behaves like a matrix). 
+As you will learn in Chapter \@ref(subsetting), you can subset a data frame or a tibble like a 1D structure (where it behaves like a list), or a 2D structure (where it behaves like a matrix). 
 
 In my opinion, data frames have two undesirable subsetting behaviours:
 
 * When you subset columns with `df[, vars]`, you will get a vector if `vars`
   selects one variable, otherwise you'll get a data frame. This is a frequent 
   source of bugs when using `[` in a function, unless you always remember to 
-  do `df[, vars, drop = FALSE]`.
+  use `df[, vars, drop = FALSE]`.
   
 * When you attempt to extract a single column with `df$x` and there is no 
   column `x`, a data frame will instead select any variable that starts with
   `x`. If no variable starts with `x`, `df$x` will return `NULL`. This makes
   it easy to select the wrong variable or to select a variable that doesn't
   exist.
 
-Tibbles tweak these behaviours so that `[` always returns a tibble, and `$` doesn't partial match, and warns if it can't find a variable (this is what makes tibbles surly).
+Tibbles tweak these behaviours: `[` always returns a tibble; `$` doesn't do partial matching and returns a warning if it can't find a variable (this is what makes tibbles surly).
```

Comment:
A `,` instead of `;` would feel more natural to me

Reply:
"Tibbles tweak these behaviours so that a `[` always returns a tibble, and a `$` doesn't do partial matching and warns if it can't find a variable (this is what makes tibbles surly)."

Reply:
:+1: